src/libraries/System.Private.CoreLib/src/System/Number.BigInteger.cs

```diff
@@ -342,7 +325,6 @@ public static void Add(ref BigInteger lhs, ref BigInteger rhs, out BigInteger re
                 int smallLength = small._length;
 
                 // The output will be at least as long as the largest input
-                result = new BigInteger(0);
```

Comment:
Doesn't `result` still need to be zero initialized or is this bypassed because `_blocks` is a fixed sized buffer?

Reply:
Correct. Initialization is not enforced on fixed buffers.